docs/apm/apm-alerts.asciidoc

```diff
@@ -107,7 +107,7 @@ From this page, you can disable, mute, and delete APM alerts.
 [[apm-alert-more-info]]
 === More information
 
-See {kibana-ref}/alerting-getting-started.html[alerting and actions] for more information.
+See {kibana-ref}/alerting-getting-started.html[Kibana Alerting] for more information.
```

Comment:
Should we use `{kib}`? I'm not sure why there is a variable for it, unless it needs translation in some regions/locales?

Reply:
Updated in https://github.com/elastic/kibana/pull/119314/commits/f8dcbce022353d226332a67391e98c9d37159051